docs/architecture/testing/psa-storage-format-testing.md

```diff
@@ -4,7 +4,9 @@
 
 The PSA crypto subsystem includes a persistent key store. It is possible to create a persistent key and read it back later. This must work even if Mbed TLS has been upgraded in the meantime (except for deliberate breaks in the backward compatibility of the storage).
 
-The goal of this document is to define a test strategy for the key store that not only validates that it's possible to load a key that was saved with the version of Mbed TLS under test, but also that it's possible to load a key that was saved with previous versions of Mbed TLS.
+The goal of this document is to define a test strategy for the key store that not only validates that it's possible to load a key that was saved with the version of Mbed TLS under test, but also that it's possible to load a key that was saved with previous versions of Mbed TLS using the same storage format version.
+
+Reading or converting data that was created with an older storage format version is out of scope of this document.
```

Comment:
You mean rather "reading and converting" ? Reading only could be interpreted as loading and "Keeping up with storage format evolution" is about loading files in an old format.

Reply:
Oops, I actually didn't mean to add this sentence here at all. I started out doing that, but realized that reading old files _is_ in scope, and is covered in “Keeping up with storage format evolution”. So I meant to instead add a reference to that section to address [your comment earlier today](https://github.com/ARMmbed/mbedtls/pull/3962#discussion_r5599845770) but I forgot to remove that sentence from the introduction. I'll amend the last commit.

Reply:
Conversion could be in scope, but it isn't addressed because we don't have any plans for conversion yet. The test strategy around conversion would be heavily dependent on how the conversion is supposed to work.